### ISSUE TITLE
Typescript: React project fails to compile

### ISSUE THREAD

**Issue by jclrosa** [moltin/js-sdk#239]
I've installed the js-sdk package in my react app. I am using typescript and realized that the package have the types file `moltin.d.ts` alredy inside it.

However, when launching the project it fails to compile with the following error:
`error TS7010: 'constructor', which lacks return-type annotation, implicitly has an 'any' return type.`

I've been able to fix the issue, by adding the type into all the constructors that are used in the `moltin.d.ts` file.

Basically I turned `constructor(options: ConfigOptions)` into `constructor(options: ConfigOptions): void` and did the same on all the constructors of the file.

So resuming, I just wanted to know if this is some issue on the types file of the package, or if it something that is wrong on my setup.

Package-version: "@moltin/sdk": "^3.16.0"
React-version: "react": "^16.0.0"

**Comment by phidah** [comment]
Same issue here.

**Comment by johnrjj** [comment]
Yeah, the types have not been working for us either, we use a modified local version of the types (copied the moltin.d.ts and have been adjusting as needed).

**Comment by notrab** [comment]
@richardscarrott is this perhaps something you could assist with here?

**Comment by Gounlaf** [comment]
> Yeah, the types have not been working for us either, we use a modified local version of the types (copied the moltin.d.ts and have been adjusting as needed).

Hi @johnrjj,

Can you please share your solution?

**Comment by RyanKirby89** [comment]
Typescript support has been updated in the SDK, there should be no more compile errors.
